docs/whats-new/staging-release-notes.md

```diff
@@ -2,14 +2,70 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 15 March 2023
+
+Frontend version: 1.0.0-20230314.0632 | Backend version: 1.54.0-230309.0902
```

Comment:
is this the correct version? in DEV the backend version is 1.54.1-230312.1328

Reply:
@JLeeChan yinqi mentioned this:
 apologies, there's update to FE version, latest is FE 1.0.0-20230314.0632 BE 1.54.0-230309.0902